packages/web-components/fast-components/src/menu-item/menu-item.styles.ts

```diff
@@ -50,6 +51,33 @@ export const menuItemStyles = (context, definition) =>
         border: calc(${focusOutlineWidth} * 1px) solid transparent;
     }
 
+    :host(.indent-0) {
```

Comment:
just tossing it out - does `align-0` make more sense here than indent, or do we feel indent is sufficiently descriptive of what's happening? Would it help to document this behavior as part of the Foundation Readme so that folks understand the intent when they are extending foundation?